app/src/features/standards/components/StandardsToolbar.tsx

```diff
@@ -0,0 +1,64 @@
+import Icon from '@mdi/react';
+import Button from '@mui/material/Button';
+import ToggleButton from '@mui/material/ToggleButton/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import Typography from '@mui/material/Typography';
+import React, { SetStateAction } from 'react';
+import { IStandardsPageView, StandardsPageView } from '../StandardsPage';
+
+interface IStandardsToolbar {
+  views: IStandardsPageView[];
+  currentView: StandardsPageView;
+  setCurrentView: React.Dispatch<SetStateAction<StandardsPageView>>;
+}
+
+/**
+ * Toolbar for setting the standards page view
+ *
+ * @param props
+ * @returns
+ */
+export const StandardsToolbar = (props: IStandardsToolbar) => {
+  const { views, currentView, setCurrentView } = props;
+
+  return (
+    <>
+      <Typography component="legend">Data types</Typography>
+      <ToggleButtonGroup
+        orientation="vertical"
+        value={currentView}
+        onChange={(_event: React.MouseEvent<HTMLElement>, view: StandardsPageView) => {
+          if (view) {
```

Comment:
Nit: Is view ever undefined?

Reply:
Yes, if you click on an active view, view will be undefined and no tab will be selected. Wrapping with an if check makes it so there is always a view defined.

Reply:
Roger. We should update the typing so it is more clear. 
```js view: StandardsPage | null```